lib/plugins/wallet/bitcoind/bitcoind.js

```diff
@@ -62,7 +62,7 @@ function estimateFee () {
 
 function calculateFeeDiscount (feeMultiplier) {
   // 0 makes bitcoind do automatic fee selection
-  const AUTOMATIC_FEE = 0
+  const AUTOMATIC_FEE = process.env.NODE_ENV === 'development' ? 0.01 : 0
```

Comment:
Regtest transactions won't go through without this?

Reply:
Nope, just because there's not enough transaction volume to let the blockchain automatically decide how much fee to allocate. So it's easier to just define a static fee value on regtest environments